build/common.ps1

```diff
@@ -205,7 +201,19 @@ Function Install-DotnetCLI {
 
         if ($Version -eq 'latest') {
             #Get the latest specific version number for a certain channel from url like : https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/3.0.1xx/latest.version"
-            $httpGetUrl = "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/" + $Channel + "/latest.version"
+            $latestVersionLink = "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/" + $Channel + "/latest.version"
+            $latestVersionFile = Invoke-RestMethod -Method Get -Uri $latestVersionLink
+
+            $stringReader = New-Object -TypeName System.IO.StringReader -ArgumentList $latestVersionFile
+            [int]$count = 0
+            while ( $line = $stringReader.ReadLine() ) {
+                if ($count -eq 1) {
+                    $expectedVersion = $line.trim()
+                }
+                $count += 1
+            }
+
+            $httpGetUrl = "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/" + $expectedVersion + "/productVersion.txt"
             $versionFile = Invoke-RestMethod -Method Get -Uri $httpGetUrl
```

Comment:
Do we remember why we're downloading `latest.version` and `productVersion.txt` ourselves?  It feels like something that `dotnet-install.ps1` should be doing, and not be our responsibility.

If we remember why we're doing this, can we add a comment explaining why?

Reply:
Sorry that I didn't add a comment in my previous PR before.
This is to get the specific version for 'latest' version.
When there are multiple versions of SDK,
1. We check if the specific version is installed successfully by checking the existence of the path with specific version at https://github.com/NuGet/NuGet.Client/blob/dev/build/common.ps1#L238-L240
2.For people who works on local machine, if `-force` is not specified when running configure.ps1, and if any version of SDK is already installed, it will not download and install again https://github.com/NuGet/NuGet.Client/blob/dev/build/common.ps1#L231-L233
So it save a little bit of time.

Reply:
Do we want a comment here?

Reply:
If it's not that obvious, perhaps adding a comment to make it clear to everyone who will maintain it in future?

Reply:
Added a comment.